### DISCUSSION TITLE
Any plan to upgrade FLOWUnsteady as FLOWVPM becomes open-source?

### DISCUSSION THREAD

**Discussion by dingzhw** [byuflowlab/FLOWUnsteady#35] [Q&A] [answered]
It is really wonderful that `FLOWVPM` becomes open source. 
I've tried some of `FLOWUnsteady` examples using `FLOWVPM` in /tag/2.0.0 and get some fantastic plots with `paraview`. The `julia` version for evaluation is 1.7.2 and the calculating environment is Windows10 with WSL2(Ubuntu22.04). 
Almost everything works well thanks to the efficient organization of the codes. 
The "*Reformulated Vortex Particle Method*" looks more appealing. I wonder if there is any plan to upgrade `FLOWUnsteady` to include the changes of `FLOWVPM` from /tag/2.0.0 to /tag/3.0.0?

**Comment by EdoAlvarezR** [comment, marked as answer]
Sorry for the late reply. This has been implemented in the [v2.3.0 release](https://github.com/byuflowlab/FLOWUnsteady/releases) =]
